standalone-ha.xml

```diff
@@ -609,7 +604,6 @@
             <server name="default-server">
                 <ajp-listener name="ajp" socket-binding="ajp"/>
                 <http-listener name="default" socket-binding="http" redirect-socket="https" proxy-address-forwarding="true" enable-http2="true"/>
-                <https-listener name="https" socket-binding="https" proxy-address-forwarding="true" security-realm="ApplicationRealm" enable-http2="true"/>
```

Comment:
What does removing this line do? I'm not familiar with JBoss, but it looks like the http-listener on the previous line might redirect requests to this listener, or have I misunderstood?

Reply:
I was just following the documentation on how to configure the SSL here: https://www.keycloak.org/docs/latest/server_admin/#_x509 and taking out what it said to add in, ie working backwards. Will remove the redirect from the http default listener